nerdlandbot/commands/church.py

```diff
@@ -0,0 +1,70 @@
+import typing
+import asyncio
+from datetime import datetime, timedelta
+import discord
+from discord.ext import commands
+
+from nerdlandbot.translations.Translations import get_text as translate
+from nerdlandbot.helpers.TranslationHelper import get_culture_from_context as culture
+from nerdlandbot.helpers.channel import get_channel
+from nerdlandbot.commands.GuildData import get_all_guilds_data, get_guild_data, GuildData
+
+
+
+class Kerk(commands.Cog, name="Church"):
+    def __init__(self, bot: commands.Bot):
+        self.bot = bot
+
+    @commands.command(name="16uchurch", hidden=True)
```

Comment:
Maybe we can add some aliases since people will most likely forget what the name is? Like 16u_church, church, kerk, 16u_kerk, 16ukerk, ...

Reply:
I'll merge this and open an issue/do a quick PR.